OrbitLinuxTracing/Utils.h

```diff
@@ -76,6 +76,27 @@ inline int GetNumCores() {
   return 1;
 }
 
+#if defined(__x86_64__)
+
+#define READ_ONCE(x) (*(volatile typeof(x)*)&x)
+#define WRITE_ONCE(x, v) (*(volatile typeof(x)*)&x) = (v)
+#define barrier() asm volatile("" ::: "memory")
```

Comment:
Here and below: Is there a specific reason why you write the names with lowercase letters? Most other macros seem to be defined with UPPERCASE, which I find useful as I immediately know it's a macro. I assume it's because these are the standards names for these operations, so probably better to keep this as-is.

Reply:
It's to match what we see in kernel code, yes.

Reply:
Thanks for the link btw.